docs/contributing/adding-images-to-the-wesite.md

```diff
@@ -0,0 +1,32 @@
+# Adding images to the design system website
+When adding or amending guidance on the design system website and you want to add an image, it's important to be considerate of the impact images have on the website. They can have implications for web performance and accessibility if used improperly.
+
+Before reading the following, also consult our [image guidance](https://design-system.service.gov.uk/styles/images/) for details on when it's appropriate to use images and how best to use them.
+
+## Does your content really need images?
+Generally, we would avoid adding new images to our content.
+
+In patterns and components, we should be using code examples when we want to convey what our pattern or component will look like. Avoid using screenshots of patterns or components wherever possible.
+
+## Can your image be an SVG?
+SVG's are usually better for web performance as they scale with different screen sizes and load faster than most image formats of the same size. If your image is a "graphic" (in other words it isn't a photo or a screenshot) it should be converted to an SVG and added to the page directly.
+
+This doesn't however mean that SVG's can't also have a negative impact on performance if their file size is unnecessarily large. Consider using a compression service such as [SVGOMG](https://jakearchibald.github.io/svgomg/) to ensure your SVG is as performant as possible.
+
+## Ensure your image is an appropriate file size
+If your image can't or shouldn't be an SVG is instead a "standard" file type eg: PNG, JPEG etc, avoid adding very large file size images to the website.
+
+The simplest way to avoid this is is to compress your images. There are numerous compression tools such as [ImageOptim](https://imageoptim.com/mac) which can handle this for you.
```

Comment:
For PNGs especially, limiting the palette to 256 colours (8 bit) will probably save more than a compression tool

Reply:
I don't think this is correct. PNGs are *highly* compressible / optimisable, especially for things like screenshots. I'd absolutely suggest teams run PNGs through an optimiser before reducing the colour palette.

Reply:
Sorry yep, I'd do both where applicable (compression and/or reduced colours)

Reply:
It depends what the PNGs are images of course. PNGs are suited to things like diagrams or screenshots, but not images / photographs. I suspect reducing colours won't help a huge amount for diagrams and screenshots as there won't be many colours in the first place. If the PNG is a photo, then they should be resaving it as JPEG rather than reducing colours and keeping it as PNG.

Reply:
Do we think there's value in adding a section above this one specifying that PNG and JPEG have different use cases?

Reply:
Potentially, but then we're back to `<picture>` from https://github.com/alphagov/govuk-design-system/pull/2410#discussion_r1104406949?

Where both AVIF and WebP are better suited than JPEG in modern browsers

```html
<picture>
  <source type="image/webp" srcset="example.webp, example@2x.webp 2x, example@3x.webp 3x">
  <img src="example.jpg" alt="A photo better suited to WebP compression, but JPEG will do.">
</picture>
```

Newer formats have such huge support I'm not sure we should document all the quirks of each format:
https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Image_types

But all image formats should be appropriately chosen and be run through compression tools

(Not necessarily just PNG, JPEG or SVG)